app/assets/stylesheets/views/_homepage.scss

```diff
@@ -102,16 +102,20 @@
   padding: 0;
 
   @include govuk-media-query($from: desktop) {
-    @include columns($items: 10, $columns: 2, $selector: "li", $flow: column);
+    width: 66.66%;
+    columns: 2;
+    column-gap: 15px;
   }
+}
 
-  & > li {
-    margin: 0 0 govuk-spacing(4) 0;
-    @include govuk-font($size: 19);
-
-    @include govuk-media-query($from: desktop) {
-      margin-bottom: govuk-spacing(2);
-    }
+.homepage-most-active-list__item {
+  margin: 0 0 govuk-spacing(4) 0;
+  @include govuk-font($size: 19);
+  @include govuk-media-query($from: desktop) {
+    break-inside: avoid-column;
+    margin-bottom: 0;
+    padding-top: 0.1875rem;
+    padding-bottom: calc(#{govuk-spacing(2)} - 0.1875rem);
```

Comment:
You've put a really thorough explanation in the git commit message - could the explanation also be put here as a comment to make it easier to find without using `git blame`?

Reply:
I've updated the PR description with the same commit message, is that OK?

Reply:
I've added a more detailed comment as requested.